crates/bevy_gltf/src/loader/gltf_ext/mesh.rs

```diff
@@ -1,13 +1,17 @@
 use bevy_mesh::PrimitiveTopology;
 
-use gltf::mesh::{Mesh, Mode, Primitive};
+use gltf::{
+    mesh::{Mesh, Mode},
+    Material,
+};
 
 use crate::GltfError;
 
-pub(crate) fn primitive_name(mesh: &Mesh<'_>, primitive: &Primitive) -> String {
+pub(crate) fn mesh_name(mesh: &Mesh<'_>, material: &Material) -> String {
```

Comment:
```suggestion
pub(crate) fn primitive_name(mesh: &Mesh<'_>, material: &Material) -> String {
```

I'd suggest keeping it as `primitive_name`. Feels more appropriate when it's naming the entity that corresponds to a primitive.